lib/stackup/rake_tasks.rb

```diff
@@ -75,42 +78,69 @@ def define
       end
     end
 
-    # Options to "stackup up".
-    #
-    class DataOptions
+    # A flag with optional argument that will be passed to stackup
+    class DataOption
 
-      def initialize
-        @options = {}
+      def initialize(flag, argument)
+        @flag = flag
+        @argument = argument if argument
```

Comment:
`@argument` isn't an option field to this method, so no need for the post-if clause here.